test/compilable/ice10598.d

```diff
@@ -1,3 +1,9 @@
 // EXTRA_SOURCES: imports/ice10598a.d imports/ice10598b.d
+/* TEST_OUTPUT:
+---
+compilable/imports/ice10598a.d(5): Deprecation: module imports.ice10598b is not accessible here, perhaps add 'static import imports.ice10598b;'
+compilable/imports/ice10598a.d(5): Deprecation: module imports.ice10598b is not accessible here, perhaps add 'static import imports.ice10598b;'
+---
+*/
```

Comment:
maybe move this to fail_compilation and add `REQUIRED_ARGS: -de`

Reply:
Done.